### DISCUSSION TITLE
Jólajólajól Biskupsreglunnar - 2023/11/0

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#104] [vefsida-ummæli]
# Jólajólajól Biskupsreglunnar - 2023/11/0

Biskupsreglan hélt upp á vor jól í skólanum síðast liðnu daga og það var mjög skemmtilegt

https://www.biskupsreglan.com/frettir/Jolajolajol

**Comment by TJ478** [comment]
Frábær jól...
